modules/odc-adding-a-GitHub-repository-containing-pipelines.adoc

```diff
@@ -51,3 +51,8 @@ image::Git-access-token-secret.png[]
 . Navigate back to the developer console and click *Add*.
 . Read the details of the steps that you have to perform and click *Close*.
 . View the details of the repository you just created.
+
+[NOTE]
+====
+When importing an application using *Import from Git* and the Git repository has a '.tekton` directory, you can configure `pipelines-as-code` for your application. For more information see, xref:../../applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc#odc-creating-applications-using-the-developer-perspective[Creating applications using the Developer perspective].
```

Comment:
Modules cannot contain xrefs. Maybe add Additional Information section in assembly after this module is imported

Reply:
@gabriel-rh sure, I'll keep this in mind for future PRs as well. I'll remove the xref from here.